nrm_django/grants/tests/test_views.py

```diff
@@ -64,3 +69,42 @@ def test_logged_in_uses_correct_template(self):
                 "grants/update.html",
                 f"The template grants/update.html was not used for grant {grant.cn}.",
             )
+
+    # These next 5 tests could be refactored into one containing all of the assertions.
```

Comment:
One way to DRY these out is to sub-class GrantUpdateViewTest, extend the `setUp` method with these five repeated lines and then just write the tests as assertions about `self.response`.

Reply:
Thanks, I'll do that and update this PR.